### DISCUSSION TITLE
pages/Fault-Analyzer-for-zOS/FA-host_doc_output/idiug88

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#45] [Q&A]
# pages/Fault-Analyzer-for-zOS/FA-host_doc_output/idiug88

This section tells you how to customize Fault Analyzer for your particular installation, and how to set up global default options. There are times when you might want to set or change an option just ...

https://github01.hclpnp.com/pages/Fault-Analyzer-for-zOS/FA-host_doc_output/idiug88.html

**Comment by LouIentile** [comment]
A new comment
